src/TerrainRenderer.cpp

```diff
@@ -1071,30 +1073,29 @@ void TerrainRenderer::AltitudeChanged(const MapPoint pt, const GameWorldViewer&
     UpdateVertexColor(pt, gwv);
 
     for(unsigned i = 0; i < 6; ++i)
-        UpdateVertexColor(gwv.GetNeighbour(pt, i), gwv);
-
+        UpdateVertexColor(gwv.GetNeighbour(pt, Direction::fromInt(i)), gwv);
 
     // und für die Ränder
     UpdateBorderVertex(pt, gwv);
 
     for(unsigned i = 0; i < 6; ++i)
-        UpdateBorderVertex(gwv.GetNeighbour(pt, i), gwv);
+        UpdateBorderVertex(gwv.GetNeighbour(pt, Direction::fromInt(i)), gwv);
 
     // den selbst sowieso die Punkte darum updaten, da sich bei letzteren die Schattierung geändert haben könnte
     UpdateTrianglePos(pt, gwv, true);
     UpdateTriangleColor(pt, gwv, true);
 
     for(unsigned i = 0; i < 6; ++i)

```

Comment:
Direction is a scoped enum. It is (currently) not possible to iterate over it directly (But should be implemented later)
The fromInt method is a zero-overhead funtion for converting